src/connectors/tagService.ts

```diff
@@ -150,6 +150,97 @@ export class TagService extends BaseService {
     return item
   }
 
+  /**
+   * Count of a tag's participants.
+   *
+   */
+  countParticipants = async ({
+    id,
+    exclude,
+  }: {
+    id: string
+    exclude?: string[]
+  }) => {
+    const subquery = this.knex.raw(`(
+        SELECT
+            at.*, article.author_id
+        FROM
+            article_tag AS at
+        INNER JOIN
+            article ON article.id = at.article_id
+        WHERE
+            at.tag_id = ${id}
+        ORDER BY
```

Comment:
Same as `findParticipants` but just for counting? `ORDER BY` seems unnecessary.

Reply:
Removed it.